### PR TITLE
Combine darkmode and regular logo

### LINKED ISSUE #259: Combine darkmode and regular logo
### What feature do you want to see added?

Instead of having https://github.com/jenkinsci/design-library-plugin/blob/master/logo-dark.svg and https://github.com/jenkinsci/design-library-plugin/blob/master/logo.svg, we can make use of the `prefers-color-scheme: dark` attribute to apply colors, based on the theme the user requests.

Such a functionality allows using 1 image for multiple themes, like I did it with the header logo in https://github.com/jenkinsci/jenkins/blob/master/README.md

### Upstream changes

_No response_

### PR DESCRIPTION
**Pull request by pyatizbyantsevia**
All logic moved to logo.svg. Since the paths between the repositories to logo.svg are different, the working version can be viewed on the master branch of my [fork](https://github.com/pyatizbyantsevia/design-library-plugin)
Closes #259

### Submitter checklist

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

<a href="https://gitpod.io/#https://github.com/jenkinsci/design-library-plugin/pull/263"></a>

### CODE REVIEW

File: logo.svg [hunk 1]
```diff
@@ -26,7 +26,17 @@
...
+            <path class="figures" d="M32.6137819,33.8444602 L2.54969389,33.8444602 C1.14153684,33.8444602 0,32.6769536 0,31.2367612 C0,30.7913947 0.11152861,30.3534497 0.32395522,29.9646699 L15.3559992,2.45326441 C16.0429283,1.19605782 17.5962915,0.7464252 18.825533,1.44898181 C19.237395,1.68437608 19.5773184,2.03203256 19.8074766,2.45326441 L34.8395205,29.9646699 C35.5264496,31.2218764 35.0868185,32.8105785 33.857577,33.5131351 C33.4774452,33.7303944 33.0492417,33.8444602 32.6137819,33.8444602 Z M28.1160234,17.094738 C35.2111906,13.0647288 44.1049062,14.3293322 49.8178648,20.1805375 C55.5308233,26.0317428 56.6451013,35.0173145 52.5368502,42.1065197 C48.4285992,49.195725 40.1174634,52.6290638 32.2565807,50.4843201 C24.3956981,48.3395766 18.9360944,41.1490635 18.9344262,32.9385181" stroke="#000000" stroke-width="4"></path>
         </g>
+        <style>
+            @media (prefers-color-scheme: dark) {
+                .figures {
+                    stroke: #c9d1d9;
+                }
+                .text {
+                    fill: #c9d1d9;
```
Comment (NotMyFault) [inline logo.svg:37]:
```suggestion
                    stroke: #ffffff;
                }
                .text {
                    fill: #ffffff;
```

**Review by NotMyFault — COMMENTED** [review]
Hey,

Thanks so much for the PR! I'd recommend using switching the white for darkmode users, that matches with the symbol colors in Jenkins used for darkmode.

File: logo.svg (hunk 1 above)
Comment (pyatizbyantsevia) [reply logo.svg]: Good evening!
Thank you very much for your prompt reply. This is my first open source PR. Tell me, do I need to squash my commits after each reviewer's comments every time?

**Review by NotMyFault — APPROVED** [review]
Thanks!

File: logo.svg (hunk 1 above)
Comment (NotMyFault) [reply logo.svg]: > Tell me, do I need to squash my commits after each reviewer's comments every time?

Nope, all good, we commonly squash PRs anyway.

**Pull request merged**